### DISCUSSION TITLE
Week 1 About Me Coding Assignment

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#12] [Assignments]
Read [Chapter 1: HTML](https://geobgu.xyz/web-mapping2/html.html) 

Using VS Code, and in the same `Week1` folder that you completed your lab assignment, create a new html file titled "aboutme.html"

- Make the title "About me" or something to your liking (use `<h1>` tags)
- Add a paragraph or two introducing yourself and your goals for this course (use `<p>` tags)
- Use the appropriate tags to format some of the words in *italic* or **bold** font
- Use the `<a>` tag to add a link to another web page
- Add a list with two levels, i.e., a list where each list item is also a list.
- Add an image (or two) or avatar of yourself (use `<img`> tags)
- Push you new file to your GitHub repo (follow the instructions from the lab)

**Comment by kimjee8955** [comment]
https://kimjee8955.github.io/DH151/Week1/aboutme.html

**Comment by lisa58** [comment]
https://lisa58.github.io/DH151/Week1/aboutme.html

**Comment by anncong** [comment]
https://anncong.github.io/DH151/Week%201/aboutme.html

**Comment by eliaranita1** [comment]
https://eliaranita1.github.io/DH151/week%201/aboutme.html

**Comment by derekwang99** [comment]
https://derekwang99.github.io/DH151/Week%201/aboutme

**Comment by JuliaWood1** [comment]
https://juliawood1.github.io/DH151/Week1/aboutme.html

**Comment by alicelu170** [comment]
https://alicelu170.github.io/DH151/Week%201/aboutme.html

**Comment by justisg** [comment]
https://justisg.github.io/DH151/Week1/aboutme.html

**Comment by adrianaromero819** [comment]
https://adrianaromero819.github.io/DH151/Week%201/aboutme.html

**Comment by MaryJaneHowland** [comment]
https://maryjanehowland.github.io/DH151/Week%201/aboutme.html

**Comment by hanarama** [comment]
https://hanarama.github.io/Week%201/aboutme.html

**Reply by jmtanenbaum** [reply]
your background!!

**Comment by LynnT2** [comment]
https://lynnt2.github.io/DH151/Week%201/aboutme

**Comment by kaisanwatanabe** [comment]
https://kaisanwatanabe.github.io/DH151/week%201/aboutme.html

**Comment by alekhyavittalam** [comment]
https://alekhyavittalam.github.io/DH151/Week1/aboutme.html

**Comment by ivanma9** [comment]
https://ivanma9.github.io/DH151/Week1/aboutme

**Comment by danielliuzy** [comment]
https://danielliu524.github.io/DH151/Week1/aboutme.html

**Comment by natgrace** [comment]
https://natgrace.github.io/DH151/Week1/aboutme.html

**Comment by zonaliao** [comment]
https://zonaliao.github.io/DH151/Week1/aboutme.html

**Comment by hoz-map** [comment]
https://hoz-map.github.io/DH151/Week1/aboutme.html

**Comment by jomh17** [comment]
https://jomh17.github.io/DH151/WEEK%201/aboutme.html

**Comment by ainsleighdouglas** [comment]
[https://ainsleighdouglas.github.io/DH-151/Week1/aboutme.html](url)

**Reply by yohman** [reply]
The link is not working for me. Can you repost?

**Reply by ainsleighdouglas** [reply]
[https://ainsleighdouglas.github.io/DH-151/Week1/aboutme.html](url) Sorry about that!

**Comment by ryanorlando** [comment]
https://ryanorlando.github.io/Dh151/Week1/aboutme.html

**Comment by sebdills** [comment]
https://github.com/sebdills/DH151/blob/1133a82f15f0c865674e8276dfbccb062b220041/Week%201/aboutme.html

**Reply by yohman** [reply]
Can you provide the github pages url?

**Comment by lisalou36** [comment]
https://lisalou36.github.io/DH151/week%201/aboutme2/aboutme2.html

**Comment by jmtanenbaum** [comment]
https://jmtanenbaum.github.io/DH151/week1/aboutme.html

**Comment by yjwsb233** [comment]
https://github.com/yjwsb233/DH151/blob/main/Week1/aboutme.html

**Reply by yohman** [reply]
Can you provide the GitHub pages url?

**Reply by yjwsb233** [reply]
https://yjwsb233.github.io/DH151/Week1/aboutme.html

**Comment by kaicolorado** [comment]
https://github.com/kaicolorado/DH151/blob/main/Week%201/aboutme.html

**Reply by yohman** [reply]
Can you provide the GitHub pages url?

**Reply by kaicolorado** [reply]
https://kaicolorado.github.io/DH151/Week%201/aboutme.html

**Comment by ec028** [comment]
https://ec028.github.io/DH151/Week1/aboutme.html

**Comment by kmcinerny** [comment]
https://kmcinerny.github.io/DH151/Week1/aboutme.html

**Comment by frida-barragan** [comment]
https://frida-barragan.github.io/DH151/Week1/aboutme.html

**Comment by michelle14lee** [comment]
https://michelle14lee.github.io/DH151/Week1/aboutme.html

**Comment by tayleemae** [comment]
https://github.com/tayleemae

**Comment by alexthegreat1** [comment]
https://alexthegreat1.github.io/DH151/Week%201/aboutme.html
